### PR TITLE
Add manifest for org.kde.kalgebra

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 
  - https://invent.kde.org/education/kalgebra/-/merge_requests/22 (merged)

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

<hr>

### Notes

Icon in title bar appears if using KDE on host system. But on Xfce, it doesn't appear.

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.kalgebra

File: eigen.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (hfiguiere) [inline eigen.json:1]: you don't need a manifest for a signle module like that. It's harder to maintain / read

File: org.kde.kalgebra.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+	"analitza.json",
```
Comment (hfiguiere) [inline org.kde.kalgebra.json:20]: same comment here: just put it all in the same .json.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.kalgebra.json (hunk 2 above)
Comment (flyingcakes85) [reply org.kde.kalgebra.json]: Sure, updated in latest commit.

**Comment by flyingcakes85** [comment]
bot, build org.kde.kalgebra

**Pull request closed without merging**